.github/workflows/publish-helm.yml

```diff
@@ -0,0 +1,56 @@
+---
+name: publish-helm-chart
+
+on:  # yamllint disable-line rule:truthy
+  workflow-dispatch:
+    inputs:
+      tag:
+        description: "Release Tag"
+        required: "true"
+        default: "0.0.0"
```

Comment:
Should this instead match the current release version? If this is a hard coded string, it'll be bumped during automatic version bumps.